app/assets/stylesheets/rad_common/rad_common.css

```diff
@@ -259,3 +267,62 @@ td.fa.security-group-td { border-style: none; }
     margin-bottom: 0;
   }
 }
+
+/* custom bootstrap styles */
+
+/* this is used because the "as: :hidden" option adds a hidden class to the form group, but that class was removed in b4. So for now without this, we're left with extra spacing around a hidden input. */
+.form-group.hidden {
+  display: none;
+}
+
+.card {
+  margin-bottom: 20px;
+}
+
+/* preserved from bootstrap 3 panel-default styles */
+/* TODO: convert views to standard card usage and remove the below */
+.card-default {
+  border-color: #ddd;
+}
+.card-default > .card-header {
+  color: #333;
+  background-color: #fff;
```

Comment:
Just curious, are we changing the color here? Just saw this as I was implementing this for better_way_ars. Our old one was more of a grey. No problem either way, just making sure this is intentional